### ISSUE TITLE
Kubemqctl will not 'get dashboard' after following quickstart

### ISSUE THREAD

**Issue by invidious9000** [kubemq-io/kubemqctl#19]
**Environment:**
minikube v1.16.0 on Microsoft Windows 10 Pro 10.0.19041 Build 19041
Kubernetes v1.20.0 on Docker 20.10.0

**Approaches attempted:**
kubectl deployment + kubemqctl create dashboard
helm operator + helm dashboard deployment

For both approaches `kubemqctl get dashboard` produces this output (namespace is default/ for helm):
```
Current Kubernetes cluster context connection: minikube
Connecting to kubernetes cluster...Current Kubernetes cluster context connection: minikube
error: no running pods available in kubemq/kubemq-dashboard.
```

When I examine the service using kubectl or kubernetes dashboard the service appears to be healthy, the pod is up and running in the expected namespace; I can connect to KubeMQ dashboard by port-forwarding Grafana port (3000).

**Comment by kubemq** [comment]
Hi,
Please join our slack channel so we can support you instantly.
Lior,

**Comment by shreyasm312** [comment]
I have same issue, and Slack link seems to be broken, Please kindly assist with this issue.
Thank you

**Comment by kubemq** [comment]
I'll look into it

**Comment by kubemq** [comment]
@invidious9000 ,
This was fix at version v3.5.1
Thank you for reporting the issue
